grails-app/conf/application.yml

```diff
@@ -60,6 +60,14 @@ quartz:
         showTriggerNames: true
         showCountdown: true
         showTickingCloud: true
+    scheduler:
+        instanceId: application.yml
+        instanceName: DefaultQuartzPool
+        skipUpdateCheck: true
+    threadPool:
+        threadCount: 2
+        # java.lang.Thread.MIN_PRIORITY == 1
+        threadPriority: 1
```

Comment:
Should set this according to the different jobs. Refresh product availability and order summary jobs should at least have regular priority, if not elevated.